### PR TITLE
Initial push for SAGEMAKER_NOTEBOOK_NO_DIRECT_INTERNET_ACCESS

### PR DESCRIPTION
**Pull request by kritijha**
I confirm these files are made available under CC0 1.0 Universal (https://creativecommons.org/publicdomain/zero/1.0/legalcode)

*Issue https://github.com/awslabs/aws-config-rules/issues/239

*Description of changes: Initial commit

### CODE REVIEW

File: python/SAGEMAKER_NOTEBOOK_NO_DIRECT_INTERNET_ACCESS/SAGEMAKER_NOTEBOOK_NO_DIRECT_INTERNET_ACCESS.py [hunk 1]
```diff
@@ -0,0 +1,410 @@
...
+
+def evaluate_compliance(event, configuration_item, valid_rule_parameters):
+    evaluations = []
+
+    sagemaker_client = get_client('sagemaker', event)
+    notebook_instance_list = get_all_notebook_instances(sagemaker_client)
+
+    #SCENARIO 1: No Amazon SageMaker notebook instances exist
+    if not notebook_instance_list:
+        return None
```
Comment (shikharj05) [inline python/SAGEMAKER_NOTEBOOK_NO_DIRECT_INTERNET_ACCESS/SAGEMAKER_NOTEBOOK_NO_DIRECT_INTERNET_ACCESS.py]: Return NOT_APPLICABLE

Comment (kritijha) [reply python/SAGEMAKER_NOTEBOOK_NO_DIRECT_INTERNET_ACCESS/SAGEMAKER_NOTEBOOK_NO_DIRECT_INTERNET_ACCESS.py]: @shikharj05 On returning None, the RDK automatically handles it and returns NOT_APPLICABLE. I'm not sure if it should be returned explicitly as "NOT_APPLICABLE"
@jongogogo any comments on this?

Comment (jongogogo) [reply python/SAGEMAKER_NOTEBOOK_NO_DIRECT_INTERNET_ACCESS/SAGEMAKER_NOTEBOOK_NO_DIRECT_INTERNET_ACCESS.py]: `None` is OK

**Comment by kritijha** [comment]
@jongogogo Good for your review.

**Pull request merged**